RELEASE_NOTES

```diff
@@ -5,6 +5,9 @@ Amazon CloudWatch Agent 1.247358.0 (2023-01-30)
 Enhancements:
 * Add log output when log files are deleted with auto_removal feature (#659)
 
+Version With Build Hash:
```

Comment:
I don't get this change. Wouldn't we want to just put this in the title section of the release notes change for the version like 
```
========================================================================
Amazon CloudWatch Agent 1.247358.0b252413 (2023-01-30)
========================================================================
```